webapp/security/views.py

```diff
@@ -123,62 +123,58 @@ def notice(notice_id):
 
 
 def notices():
-    page = flask.request.args.get("page", default=1, type=int)
     details = flask.request.args.get("details", type=str)
     release = flask.request.args.get("release", type=str)
     order_by = flask.request.args.get("order", type=str)
+    limit = flask.request.args.get("limit", default=10, type=int)
+    offset = flask.request.args.get("offset", default=0, type=int)
 
-    releases = (
-        db_session.query(Release)
-        .order_by(desc(Release.release_date))
-        .filter(Release.version.isnot(None))
-        .all()
+    # call endpopint to get all releases and notices
+    all_releases = security_api.get_releases()
+
+    api_call = security_api.get_notices(
+        limit=limit, offset=offset, details=details, release=release
     )
-    notices_query = db_session.query(Notice)
 
-    if release:
-        notices_query = notices_query.join(Release, Notice.releases).filter(
-            Release.codename == release
-        )
+    # get notices and total results from response object
+    notices_query = api_call.get("notices")
```

Comment:
couldn't this just be `notices`? `notices_query` sounds to me like it would be a string or a filter object or something.